src/marqo/_httprequests.py

```diff
@@ -38,7 +42,14 @@ def send_request(
         path: str,
         body: Optional[Union[Dict[str, Any], List[Dict[str, Any]], List[str], str]] = None,
         content_type: Optional[str] = None,
+        max_retry_attempts: Optional[int] = None,
+        max_retry_backoff_seconds: Optional[int] = None
     ) -> Any:
+        if not max_retry_attempts:
+            max_retry_attempts = 0
```

Comment:
not sure about having the defaults defined here as well as having defaults for `MARQO_OPENSEARCH_MAX_ADD_DOCS_RETRY_BACKOFF` etc. Another option is to store the defaults here under `constants.py`

Reply:
that's a good idea, I can move it there instead if that's what's prefered